drivers/rtc/qm_rtc.c

```diff
@@ -117,7 +117,7 @@ int qm_rtc_read(const qm_rtc_t rtc, uint32_t *const value)
 {
 	QM_CHECK(rtc < QM_RTC_NUM, -EINVAL);
 	QM_CHECK(value != NULL, -EINVAL);
-	*value=QM_RTC[rtc].rtc_ccvr;
+	*value = QM_RTC[rtc].rtc_ccvr;
```

Comment:
@jeez I don't know how to achieve this without deleting the whole branch and making new PR. Everything I've tried leaves extra commits on top of what's already in.